src/service/trivia.py

```diff
@@ -5,7 +5,50 @@ class TriviaService:
+            'When going to the next stage of Deep Dives, turret ammo is automatically restocked even if the Engineer hasn\'t resupplied in the prior stage.',
+            'The "Dance Moves" statistic is purely cosmetic; while there are six different animations that the Hologram can play, none of them are any better or worse at attracting enemies.',
+            'The Shredder Swarm can be used as a cheap alternative way to break enemy armor as its high 200% armor break and long duration allows the Players to severely expose hard-hitting enemies like Praetorians and Glyphid Slashers.',
+            'The "Lead Storm" Powered Minigun fires depleted uranium according to one of the Gunner\'s class selected voice lines.',
+            'The "Hurricane" Guided Rocket System\'s weapon prototype used an early version of the Fuel Canister model.',
+            'By looking down the barrel of the "Bulldog" Heavy Revolver, it is possible to see that the bullet is a a hollow point round.',
+            'The "Bulldog" Heavy Revolver model is completely static, the hammer and cylinder do not move at all even when firing.',
+            'When the 7 in "BRT7" is interpreted as a second T, it can be read as "BRTT". This is an onomatopoeia of the weapon\'s firing sound.',
+            'The Tactical Leadburster has five barrels to shoot out from. It is unknown how it fit that much bullets within such a small profile.',
+            'In earlier versions of the game, the Deepcore GK2 had a much more realistic design with a longer barrel, smaller foregrip, and a more shapely stock.',
+            'Mission Control expresses fondness for the M1000 when unlocking it via its corresponding Weapon License Upgrade assignment. He calls it a "good choice", and it is the only weapon he does not express bemusement or caution with.',
+            'The M1000 Classic is based on the American M1 Garand semi-automatic rifle, and includes the trademark "ping" when ejecting spent clips.',
+            'The DRAK-25 Plasma Carbine\'s weapon prototype used the Deepcore 40mm PGL model.',
+            'The Special Powder\'s effect of Jury-Rigged Boomstick is similar to Team Fortress 2\'s Force-a-Nature knockback jump, which is also used by the game\'s Scout class.',
+            'The "Double Trigger" upgrade of Jury-Rigged Boomstick not only increases your fire rate by 6x, it also decreases reload time by 0.55 seconds, making it almost always better than quickfire ejector.',
+            'The name of Zhukov NUK17 is likely a reference to Georgy Zhukov, a Soviet general during World War 2. The word "zhuk" or "жук" in russian translates to "bug", which could make sense in the context of the game.',
+            'Some weapon and upgrade descriptions reference R&D (which is short for Research and Development). This is the department of DRG responsible for creating new weapons and equipment for the Dwarves to use.'
```

Comment:
The formatting looks good to me. I'm curious, where did you find these facts?

Reply:
That all facts come from [DRG Wiki](https://deeprockgalactic.fandom.com/wiki/Deep_Rock_Galactic_Wiki)

Reply:
Excellent, thanks!